### PR TITLE
Trunk-3422(new privileges which are not added by default)

### PR DESCRIPTION
**Pull request by puevigreven**
Adding a liquibase changeset that inserts the "View Encounter Provider" and "View Visits" privileges for roles that have "View Encounter".

### CODE REVIEW

**Comment by puevigreven** [comment]
@dkayiwa  have i got the indentation correct?

**Comment by dkayiwa** [comment]
@puevigreven the indention is correct! :)
I thought you would do a separate changeset for each privilege.

**Comment by puevigreven** [comment]
@dkayiwa while working with git , i deleted the old PR for this trunk by mistake. (Still learning git). 
what refining does this code wants? 
:)

**Comment by dkayiwa** [comment]
@puevigreven can you do a separate changeset for each privilege?

**Comment by puevigreven** [comment]
@dkayiwa i made the separate changeset for each. :)

**Comment by dkayiwa** [comment]
One last thing, can you put these changesets at the end of the file, as we usually do?

File: api/src/main/resources/liquibase-update-to-latest.xml [hunk 1]
```diff
@@ -8946,4 +8946,61 @@
...
+                    <sqlCheck expectedResult="0">
+                    SELECT count(*) FROM role_privilege  WHERE privilege='View Encounters';
+                    </sqlCheck>
+                </not>
+            </preConditions>
+            <comment>
+                Add "View Visits" privilege to the roles having "View Encounter"
+            </comment>
+                <sql>
+                    INSERT INTO privilege (privilege,description)
```
Comment (puevigreven) [inline api/src/main/resources/liquibase-update-to-latest.xml]: @dkayiwa suppose View Visit does not exist in privilege table(this implies that View Visits does not exists in role_privilege table) and "View Encounters" Exist in role_privilege i.e. some role has "View Encounters" privilege then this code first adds "View Visits" in privilege tables if it already does not exist and then it add "View Visits" privilege to the role which has 'View Encounter " privilege if this role does not have "View Visits" already!

Comment (dkayiwa) [inline api/src/main/resources/liquibase-update-to-latest.xml]: @puevigreven after having looked at PrivilegeConstants, "View" needs to be changed to "Get".

**Comment by puevigreven** [comment]
@dkayiwa one record. i.e. Get Encounter Provider. 
Since Get Visits is already in privilege table and it cannot be removed from the table.

File: api/src/main/resources/liquibase-update-to-latest.xml [hunk 2]
```diff
@@ -8946,4 +8946,54 @@
...
+                Add "Get Encounter Provider" privilege to the roles having "View Encounter"
+            </comment>
+                <sql>
+                    INSERT INTO privilege (privilege,description)
+                    SELECT 'Get Encounter Provider','Able to view Encounter Provider'
+                    FROM privilege
+                    WHERE NOT EXISTS (
+                    SELECT * 
+                    FROM privilege 
+                    WHERE privilege='Get Encounter Provider') limit 1;
```
Comment (puevigreven) [inline api/src/main/resources/liquibase-update-to-latest.xml]: @dkayiwa i added the limit by which only one record will be inserted. 
i am sorry for the mistake. :) 
i'll be more careful next time.

**Comment by puevigreven** [comment]
@dkayiwa the code which i have revised inserts only one record!

**Comment by puevigreven** [comment]
@dkayiwa  please review this :)

**Comment by dkayiwa** [comment]
@puevigreven this has merge conflicts. Or did you just redo a new one?

**Comment by puevigreven** [comment]
closing because of merge conflicts and main repo was deleted.

**Pull request closed without merging**